sys-apps/baselayout/baselayout-2.4.1.ebuild

```diff
@@ -144,6 +144,7 @@ src_prepare() {
 			-e "/PATH=.*\/bin/s|\"$|:/usr/bin:/bin\"|" \
 			etc/profile || die
 		sed -i -r \
+			-e "s@:/bin@:${EPREFIX}/bin@g" \
```

Comment:
https://dev.gentoo.org/~ulm/pms/head/pms.html
Check out `D`, `ED`, `ROOT`, `EROOT` and then `EPREFIX`